python/nano/src/bigdl/nano/pytorch/amp/bfloat16.py

```diff
@@ -160,7 +167,11 @@ def _load(path, model):
         state_dict = torch.load(checkpoint_path)
         model.eval()
         model.load_state_dict(state_dict)
-        return BF16Model(model, channels_last=status['channels_last'])
+        thread_num = None
+        if status["thread_num"] is not None:
```

Comment:
apologize for not revising this file :(

We need
```python
if status["thread_num"] is not None and status['thread_num'] != {}:
```
since `yaml` will parse a `{}` rather than `None` for `null` in yml file.

Reply:
Done